lib/screen/club_detail/club_detail_screen.dart

```diff
@@ -61,7 +64,16 @@ class ClubDetailScreenState extends State<ClubDetailScreen> with BackNavigatorMi
                           Row(
                             crossAxisAlignment: CrossAxisAlignment.start,
                             children: [
-                              Column(children: [
+                              Column(
+                                crossAxisAlignment: CrossAxisAlignment.start,
+                                children: [
+                                ConditionalShower<List<Activity>>(
+                                  data: viewModel.activities,
+                                  builder: (context, activities) => ActivityOverview(
+                                    title: activities[viewModel.selectedActivity].name,
```

Comment:
I would give an `ActivityOverview` just the `activity` and the title can then be derived from the activity. Now you ask the view model twice for the same activity